nixos/modules/services/monitoring/prometheus/exporters/py-air-control.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.py-air-control;
+
+  py-air-control-exporter-env = pkgs.python3.withPackages (pyPkgs: [
+      pyPkgs.py-air-control-exporter
+  ]);
+
+  workingDir = "/var/lib/${cfg.stateDir}";
+
+in
+{
+  port = 9896;
+  extraOpts = {
+    deviceHostname = mkOption {
+      type = types.str;
+      example = ''"192.168.1.123"'';
```

Comment:
Was the multi-line string intended here? In the test it is just a simple string without double quotes.

Reply:
Oh, that's an accident. Converted to `example = "192.168.1.123";`.